.gitignore

```diff
@@ -102,3 +103,10 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# index checkpoints | to be removed after POC is complete
```

Comment:
Why are they to be removed after PoC is complete? 
We should never push model-data to repo?

Reply:
Sounds good, will change the comment